clang/docs/APINotes.rst

```diff
@@ -195,6 +195,52 @@ declaration kind), all of which are optional:
       SwiftReleaseOp: immortal
       SwiftRetainOp: immortal
 
+:SwiftReturnOwnership:
+
+  Specifies the ownership convention of a function or method returning a C++ type
+  that has been imported as a Swift reference type using ``SwiftImportAs: reference``.
```

Comment:
> using ``SwiftImportAs: reference``

Technically, is it possible that the type is annotated via attributes and not API notes? If that is the case, I think we can just drop the "using" part as it is not necessarily true.